Resources/doc/commands/generate_controller.rst

```diff
@@ -25,37 +25,43 @@ The command can be run in a non interactive mode by using the
 Available Options
 -----------------
 
-* ``--controller``: The controller name given as a shortcut notation containing 
-  the bundle name in which the controller is located and the name of the 
-  controller. For instance: ``AcmeBlogBundle:Post`` (creates a ``PostController``
-  inside the ``AcmeBlogBundle`` bundle):
+``--controller``
+    The controller name given as a shortcut notation containing the bundle
+    name in which the controller is located and the name of the controller
+    (for instance, ``AcmeBlogBundle:Post`` creates a ``PostController`` class
+    inside the ``AcmeBlogBundle`` bundle):
 
     .. code-block:: bash
 
         $ php app/console generate:controller --controller=AcmeBlogBundle:Post
 
-* ``--actions``: The list of actions to generate in the controller class. This
-  has a format like ``%actionname%:%route%:%template`` (where ``:%template%``
-  is optional):
+``--actions``
+
```

Comment:
This description leaves 1 blank line, but others (like the `--controller` description above) don't.